pkg/networkmanager/networkmanager.jsx

```diff
@@ -117,7 +129,7 @@ const App = () => {
         return (
             <ModelContext.Provider value={model}>
                 <WithDialogs key="1">
-                    <NetworkPage privileged={superuser.allowed}
+                    <NetworkPage privileged={superuser.allowed || polkitAllowed}
```

Comment:
That's what I mean -- if you use `superuser: try`, you don't have to explicitly check `superuser` any more.